package.json

```diff
@@ -28,7 +28,7 @@
     "fs-extra": "3.0.1",
     "history": "^4.7.2",
     "html-webpack-plugin": "2.29.0",
-    "inkjs": "1.6.0",
+    "inkjs": "^1.6.0",
```

Comment:
I intentionally remove the `^` to ensure inkjs remains at version `1.6.0`, due to #22 .